File: HighPressure/Assets/Zombie.c
 using UnityEngine;
 using System.Collections;
 
 public class EnemyAI : MonoBehaviour {
     public Transform M_Player;
     // Use this for initialization
    void Start(){
     player = GameObject.FindGameObjectWithTag("Player");
     if(!player){
         Debug.Log("Make sure your player is tagged!!");
     }
 }
 void Update(){
     GetComponent<NavMeshAgent>().destination = player.transform.position;
 }
 
 
 }

File: HighPressure/Library/Collab/Original/Assets/Scripts/Zombie.cs
 using UnityEngine;
 using System.Collections;
 
 [RequireComponent (typeof(UnityEngine.AI.NavMeshAgent))]
 public class EnemyMovement : MonoBehaviour
 {
     public Transform player;
     UnityEngine.AI.NavMeshAgent nav;
 
     void Awake ()
     {
         player = GameObject.FindGameObjectWithTag ("Player").transform;
         nav = GetComponent <UnityEngine.AI.NavMeshAgent> ();
     }
 
   void Update ()
     {
             nav.SetDestination (player.position);
     }
 }